app/src/org/commcare/views/widgets/AudioWidget.java

```diff
@@ -32,18 +23,13 @@
  * @author Yaw Anokwa (yanokwa@gmail.com)
  */
 public class AudioWidget extends MediaWidget {
```

Comment:
Moved a lot of common functionalities from Audio and Video Widgets to MediaWidget